packages/e2e-tests/src/data/EducationalArticles.ts

```diff
@@ -35,17 +35,15 @@ export const faqArticles: Record<string, Record<string, string>> = {
     question: 'how-is-the-conway-ledger-era-also-called-governance-era-supported-by-lace',
     title: 'How is the Conway Ledger era (also called governance era) supported by Lace?'
   },
-  'What type of governance features are supported in Lace using the GovTool in the current SanchoNet test environment?':
-    {
-      question: 'what-type-of-governance-features-are-supported-in-lace-using-the-govtool-in-the-current',
-      title:
-        'What type of governance features are supported in Lace using the GovTool in the current SanchoNet test environment?'
-    },
+  'What type of governance features are supported in Lace using the GovTool?': {
+    question: 'what-type-of-governance-features-are-supported-in-lace-using-the-govtool-in-the-current',
```

Comment:
```suggestion
    question: 'what-type-of-governance-features-are-supported-in-lace-using-the-govtool',
```
Good time to fix this error